packages/page-staking/src/Actions/partials/Nominate.tsx

```diff
@@ -58,9 +58,7 @@ function Nominate ({ className = '', controllerId, nominating, onChange, poolId,
     }
   }, [api, onChange, poolId, selected]);
 
-  const maxNominations = api.consts.staking.maxNominations
-    ? (api.consts.staking.maxNominations as BN).toNumber()
-    : MAX_NOMINATIONS;
+  const maxNominations = MAX_NOMINATIONS;
```

Comment:
don't we want to take a value from chain if it exists? perhaps that's risky, because the name of the constant is about to change?

Reply:
I think this is this kind of constant that would never change in the future - it's a behaviour whcih is baked in our staking system deeply.